packages/closer/pages/settings/arx.tsx

```
@@ -0,0 +1,170 @@
+  const [txId, setTxId] = useState<string | null>(null);
+  const [busy, setBusy] = useState(false);
+
+  // Reset state
+  const reset = () => {
+    setSelected(null);
+    setChallenge(null);
+    setTxId(null);
+  };
+
+  // Close modal on Escape
+  useEffect(() => {
```

ESLint

[error] React Hook "useEffect" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)